Makefile.dep

```diff
@@ -416,12 +416,20 @@ ifneq (,$(filter posix_sockets,$(USEMODULE)))
   USEMODULE += vfs
 endif
 
+ifneq (,$(filter shell,$(USEMODULE)))
+  USEMODULE += uart_stdio_rx
+endif
+
 ifneq (,$(filter rtt_stdio,$(USEMODULE)))
   USEMODULE += xtimer
 endif
 
-ifneq (,$(filter uart_stdio,$(USEMODULE)))
+ifneq (,$(filter uart_stdio_rx,$(USEMODULE)))
   USEMODULE += isrpipe
+  USEMODULE += uart_stdio
+endif
+
+ifneq (,$(filter uart_stdio,$(USEMODULE)))
```

Comment:
Should we extend this to be USEMODULE += uart_stdio_tx and USEMODULE += uart_stdio_rx? Because how would the application look like? Maybe you can add tests that show how to enable/disable tx and/or rx mode of the uart?